_test/test_sqw_class/test_sqw_signal.m

```diff
@@ -20,44 +20,60 @@
             obj.sqw_2d = read_sqw(test_sqw_2d_fullpath);
             obj.save();
         end
+
+        function test_signal_fb(obj)
+            clObj = set_temporary_config_options(hor_config, 'mem_chunk_size', 1000);
+
+            w1modE = coordinates_calc(obj.sqw_2d, 'E');
+
+            fb_sqw = obj.sqw_2d.copy();
```

Comment:
this is slightly confusing as initially after the copy fb_sqw is not filebacked; that happens on the next line. given that the idiom is reasonable matlab, maybe just a comment that initially fb_sqw is not filebacked